db/data/20220705100148_backfill_hesa_updated_at_for_leading_zero_hesa_ids_on_trainees.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BackfillHesaUpdatedAtForLeadingZeroHesaIdsOnTrainees < ActiveRecord::Migration[6.1]
+  def up
+    trainees = Trainee.imported_from_hesa.where(hesa_updated_at: nil).left_joins(:hesa_student)
+
+    format_hesa_id(trainees).find_each do |trainee|
+      trainee.update_column(trainee.hesa_updated_at, trainee.hesa_student.hesa_updated_at)
+    end
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  def format_hesa_id(trainees)
+    formatted_trainees = []
+    trainees.find_each do |trainee|
+      trainee.hesa_id = trainee.hesa_id.to_i.to_s
```

Comment:
not sure if this is the best thing to do - I've written the code like this as I wanted to avoid updating the actual hesa_id in the DB. It feels like the leading zero IDs might technically be the correct ones but I'm not sure! So didn't want to permanently alter them.